MapboxNavigation/NavigationMapView.swift

```diff
@@ -176,13 +176,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             }
             
             if let location = userLocationForCourseTracking {
-                updateCourseTracking(location: location, animated: true)
-            }
-            
-            if let location = userLocationForCourseTracking, tracksUserCourse {
-                UIView.animate(withDuration: 1, delay: 0, options: [.curveLinear, .beginFromCurrentState], animations: {
-                    self.userCourseView?.center = self.convert(location.coordinate, toPointTo: self)
```

Comment:
I don't think we need to call this twice. This is called just a few lines above in `updateCourseTracking `